fuzz/fuzz_targets/raw.rs

```diff
@@ -0,0 +1,8 @@
+#![no_main]
+
+#[macro_use]
+extern crate libfuzzer_sys;
+
+fuzz_target!(|data: &[u8]| {
+    let _ = walrus::Module::from_buffer(data);
```

Comment:
Want to test the serialization here as well and if it succeeds at deserializing we immediately serialize it back out as well?

Reply:
Great idea!